board/miyoo/rootfs/etc/inittab

```diff
@@ -15,14 +15,12 @@
 
 # Startup the system
 ::sysinit:/bin/mount -t proc proc /proc
-::sysinit:/bin/mount -o remount,ro /
 ::sysinit:/bin/mkdir -p /dev/pts
 ::sysinit:/bin/mkdir -p /dev/shm
 ::sysinit:/bin/mount -a
 ::sysinit:/bin/hostname -F /etc/hostname
-::sysinit:/bin/mount /dev/mmcblk0p4 /mnt -o discard=async,noatime,nodiratime,exec
```

Comment:
default is discard=async anyway? How this 2 options differ, why commit in 5sec instead of regular 30s?

Reply:
To allow user savestate in game, wait 5s, flick the switch